docs/core-concepts/router.md

```diff
@@ -0,0 +1,146 @@
+# Router
+The router is responsible for matching routes to urls and updating the browser history. It also provides a way to navigate to routes, access the current route, and provides some routing utilities. See [Router](/api/types/Router.md) type for more information.
+
+## Creating a Router
+Create a router using the `createRouter` utility.
+
+```ts {8}
+import { createRoute, createRouter } from '@kitbag/router'
+
+const routes = [
+  createRoute(...),
+  createRoute(...),
+] as const
+
+const router = createRouter(routes)
+```
+
+## Installing
```

Comment:
is it worth repeating this from quick start?

Reply:
I think the quick start is always going to be duplicate of other docs. This page will likely grow to have more information and feel less repetitive. But I think these docs aren't repeated from quick start, quick start repeats them from here. If that makes sense.